### ISSUE TITLE
Not able to set VSCode as default editor for Git if using User Install version of VSCode

### ISSUE THREAD

**Issue by hintbw** [git-for-windows/git#1830]
- [ ] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

2.19.0 - 64-bit

```
$ git --version --build-options

git version 2.19.0.windows.1
cpu: x86_64
built from commit: d96bb8bc6c636a8869140e860e72e7bdf64bd790
sizeof-long: 4
sizeof-size_t: 8
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?
Win 10 - 64-bit
```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.17134.228]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: VisualStudioCode
Path Option: Cmd
SSH Option: OpenSSH
CURL Option: WinSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled
Enable Builtin Rebase: Disabled
Enable Builtin Stash: Disabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

**I was using the User Install version of VSCode for windows instead of the System Install version of VSCode for Windows**

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Powershell (and Git BASH)

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
I was running the installer and tried to set the default editor to VSCode
```
 - What did you expect to occur after running these commands?

To be able to set VSCode as the default editor

 - What actually happened instead?

The option to select VSCode was allowed, but the next button was disabled, which led me to believe that Git for Windows doesn't believe that VSCode is installed

**Comment by shiftkey** [comment]
Duplicate of #1801.

VSCode changed the default installer to be a per-user install, which is different to Git for Windows (it's an elevated installer, and will set the editor for all users) which is why it's blocking.

More discussion of this in #1795.

**Comment by hintbw** [comment]
Dang, I even searched for it - sorry.

**Comment by hintbw** [comment]
@shiftkey is #1741 also a duplicate of #1801 then - they all seem related (and potentially resolved by #1795 as an issue)
